src/library/scala/App.scala

```diff
@@ -37,6 +37,17 @@ import scala.collection.mutable.ListBuffer
  *  before the main method has been executed.'''''
  *
  *  Future versions of this trait will no longer extend `DelayedInit`.
+ * 
+ *  In Scala 3, `App` exists only in a limited form that does not support command line
+ *  arguments and will be deprecated in the future. If programs need to cross-build
+ *  between Scala 2 and Scala 3, it is recommended to use an explicit `main` method:
```

Comment:
it's good to give this good recommendation on how to cross-build, but there are also Scala 3 users who don't care about cross-building, and for those users, our recommendation should be to use `@main`